src/backend/core/management/commands/evaluate_search_engine.py

```diff
@@ -0,0 +1,211 @@
+"""
+Evaluate search engine performance with test documents and queries.
+"""
```

Comment:
I'm wondering if this command should be in the demo part because it is not usable with production data :thinking: